dosing_pump_controller/ReturnMenuItem.cpp

```diff
@@ -0,0 +1,15 @@
+//  Created by Andrew Florjancic on 9/22/24.
+//  Copyright © 2024 Andrew Florjancic. All rights reserved.
+#include "ReturnMenuItem.h"
+
+ReturnMenuItem::ReturnMenuItem() : MenuItem("Return", true) {}
+
+void ReturnMenuItem::showFeature() {}
```

Comment:
Not important now but let's see if we can add in some empty default implementations for the required virtual methods.